doc/source/ray-core/api/core.rst

```diff
@@ -0,0 +1,60 @@
+Core API
```

Comment:
Ray Core API

Reply:
this is a page under page "Ray Core API" so I called "Core API".